src/js.rs

```diff
@@ -434,6 +434,45 @@ extern {
     pub fn value_of(this: &Object) -> Object;
 }
 
+// WeakMap
+#[wasm_bindgen]
+extern {
+    pub type WeakMap;
+
+    /// Returns the function that created an WeakMap wrapper.
```

Comment:
Nitpick: this is currently documenting the `.constructor` property, but instead we should document the `WeakMap` constructor function itself:

> The WeakMap object is a collection of key/value pairs in which the keys are weakly referenced.  The keys must be objects and the values can be arbitrary values.